test/schemas.ts

```diff
@@ -2548,6 +2559,27 @@ export const v148Schema = {
   },
 }
 
+export const v149Schema = {
```

Comment:
Note that this "migration" is only for _test_ data and is not an _actual_ migration. A ton of our tests implicitly rely on `tokenBalances` being present (duh) and fail in unusual ways if it's empty. An actual migration would be a bit more onerous and error prone (re: the recent issue with transfer migrations) so it's an "empty" migration as stated in the PR description.